src/cluster/k8sClientHandler.go

```diff
@@ -498,3 +502,98 @@ func GetKubearmorRelayURL() string {
 	url := "kubearmor." + namespace + ".svc.cluster.local"
 	return url
 }
+
+// GetSecrets to get secrets
+func GetSecrets(k8sClient *kubernetes.Clientset, label string, name string) (*v1.Secret, error) {
+	if k8sClient == nil {
+		return nil, errors.New("k8s client not created")
+	}
+
+	secrets, err := k8sClient.CoreV1().Secrets("kube-system").Get(context.Background(), name, metav1.GetOptions{})
+
+	if err != nil {
+		log.Error().Msgf("error while getting license secrets with name: %s from kube-system namespace, error: %s", name, err.Error())
+		return nil, err
+	}
+	if secrets == nil {
+		return nil, nil
+	}
+
+	return secrets, nil
```

Comment:
```suggestion
	return secrets, nil
```
`secrets` will be `nil` in case of failure anyways.